### PR TITLE
Improve styling and responsiveness

### LINKED ISSUE #24: Styling & responsiveness improvements in the site
Following styling improvements need to be made:

1. On about us page, the heading is colliding with the upper green section, there should be some space between them.
image

2. On contact page, the boxes are not similar-sized
image

3. On services page, the boxes are not similar-sized
image

4. UI breaks on mobile view 
image

As part of this issue, each page should be tested on mobile view and checked for responsiveness issues.

### PR DESCRIPTION
**Pull request by saadamir3180**
Correct styling for pages, improve responsiveness for smaller screens
Fixes #24

### CODE REVIEW

File: components/portfolio/components/ProjectDetails.jsx [hunk 1]
```diff
@@ -65,16 +65,16 @@ export default function ProjectDetails({ portfolioTitle }) {
...
-                    <span className="effect-1">Previous Project</span>
+                    <span className="effect-1">Previous</span>
+                    <span className="effect-1">Previous</span>
                   </span>
                 </span>
               </a>
               <a href={nextProject ? `/project-details/${nextProject.title.replace(/\s+/g, '-')}` : ""} className={`nav-btn ${!nextProject ? "disabled" : ""}`}>
                 <span>
                   <span className="link-effect">
-                    <span className="effect-1">Next Project</span>
```
Comment (njmulsqb) [inline components/portfolio/components/ProjectDetails.jsx:76]: Why is this removed?

**Review by njmulsqb — CHANGES_REQUESTED** [review]

File: components/portfolio/components/ProjectDetails.jsx (hunk 1 above)
Comment (saadamir3180) [reply components/portfolio/components/ProjectDetails.jsx:76]: Was changed to fit to smaller screen sizes, else we need to decrease the size of the `Next Project` which can disturb the whole typography of the project

**Review by njmulsqb — APPROVED** [review]
